pkg/utils/utils.go

```diff
@@ -656,3 +656,34 @@ func GetPhysPortName(name string) (string, error) {
 	}
 	return "", nil
 }
+
+func GetVFPciAddressFromIndex(pfPciAddress string, vfID int) (string, error) {
```

Comment:
this should move to sriovnet

Reply:
There is private function `vfPCIDevNameFromVfIndex` https://github.com/Mellanox/sriovnet/blob/master/sriovnet_helper.go#L92
Which takes `pf-name` and `vf-index'
It would be nice to make it public and add a new function to return the VF name from index

Reply:
https://github.com/Mellanox/sriovnet/pull/36